sandpack-react/src/index.ts

```diff
@@ -12,10 +12,10 @@ export * from "./common/Layout";
 export * from "./common/Stack";
 export * from "./common/ErrorOverlay";
 export * from "./common/LoadingOverlay";
+export * from "./common/BareCodeSandboxButton";
```

Comment:
I would call this `UnstyledOpenInCodeSandboxButton`, to first keep consistency with the `OpenInCodeSandboxButton` name and to better suggest what it is